index.html

```diff
@@ -6,12 +6,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <title>Rock, Paper, Scissors - The Classic Game</title>
 </head>
 
 <body>
+    <header>
+        R.P.S. Classic!!!
+    </header>
+    <div id="rules-section-div">
+    <section id="rules-section">
+    A game of hand gestures with 3 simple rules:<br>
```

Comment:
Watch out over-using <br>. Instead try to use <ul> and css list-style-type: none;